deployment/terraform/create.go

```diff
@@ -557,15 +557,40 @@ func (t *Terraform) setupElasticSearchServer(extAgent *ssh.ExtAgent, ip string)
 		mlog.String("snapshot", snapshotName),
 		mlog.Array("indices", snapshotIndices))
 	opts := elasticsearch.RestoreSnapshotOpts{
-		WithIndices: snapshotIndices,
+		WithIndices:      snapshotIndices,
+		NumberOfReplicas: t.config.ElasticSearchSettings.InstanceCount - 1,
 	}
 	if err := es.RestoreSnapshot(repo, snapshotName, opts); err != nil {
 		return fmt.Errorf("unable to restore snapshot: %w", err)
 	}
 
-	// Wait until the snapshot is completely restored, or 30 minutes have
+	// Wait until the snapshot is completely restored, or 45 minutes have
```

Comment:
So, this was 30, now it's 45. Should we make it configurable or at least overridible through an environment variable?

Reply:
Good point. Fixed in https://github.com/mattermost/mattermost-load-test-ng/pull/749/commits/8060c61abacc2aebe30a49740728aa0db1df524b. Testing it now.